### PR TITLE
Testing To Do list: part 2

### PR DESCRIPTION
**Pull request by Camilovelag**
# Good day! 🌤️

This pull request is intended to refactor the "Awesome books" project to use classes.
It is powered by @hopeforkoami  and @Camilovelag.

## 🛠️ Changes made

✔️ Create `update.test.js` (checkbox status and description) and `clearAll.test.js` files.
✔️ Mock the HTML elements to test DOM manipulation functions.
✔️ Use the mock storage object created in Part 1 of this project to mock the localStorage updates.
✔️ Group tests using the `describe()` method.

We're expecting feedback.

Regards!  👏👏👏

### CODE REVIEW

File: __test__/update.test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import addTask from '../src/modules/add.js';
+import displayTasks from '../src/modules/display.js';
```
Comment (GzimAsani) [inline __test__/update.test.js]: - [x] Please add a test for editing the task as well

File: __test__/update.test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import addTask from '../src/modules/add.js';
```
Comment (GzimAsani) [inline __test__/update.test.js:1]: - [ ] Please add a test for editing the task as well

File: __test__/update.test.js (hunk 1 above)
Comment (GzimAsani) [inline __test__/update.test.js]: - [x]  Please add tests on this project to mock the localStorage updates

Comment (GzimAsani) [inline __test__/update.test.js:2]: - [ ]  Please add tests on this project to mock the localStorage updates

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @hopeforkoami  and @Camilovelag     :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your add and delete tests :+1:  
 
 ## Required Changes 

- [ ] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: __test__/update.test.js (hunk 1 above)
Comment (Camilovelag) [reply __test__/update.test.js]: Test to update the _description_ and  _checkbox_ status were splited from `update.test.js`. Both methods are inside the displayTasks function.

Checkbox:
https://github.com/Camilovelag/Todo_list/blob/94ff98537b3c093ba414e2318aa332c0c7b877ec/__test__/checkbox.test.js#L1-L18

Description: https://github.com/Camilovelag/Todo_list/blob/94ff98537b3c093ba414e2318aa332c0c7b877ec/__test__/description.test.js#L1-L20

Comment (Camilovelag) [reply __test__/update.test.js]: Done! 😄
LocalStorage function is nested inside `displayTasks` function.

https://github.com/Camilovelag/Todo_list/blob/94ff98537b3c093ba414e2318aa332c0c7b877ec/__mocks__/mockLocalStorage.js#L1-L9

**Review by mmsesay — APPROVED** [review]
Hi @Camilovelag & @hopeforkoami , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - Good Readme :heavy_check_mark: 
 - No linter error :heavy_check_mark: 
 - Good PR title :heavy_check_mark: 
 - Good PR summary :heavy_check_mark:  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**